core/templates/dev/head/pages/exploration_editor/translation_tab/TranslatorOverviewDirective.js

```diff
@@ -48,7 +48,7 @@ oppia.directive('translatorOverview', [
             .getAllAudioLanguageCodes();
 
           $scope.VOICEOVER_MODE = 'Record';
-          $scope.TRANSLATION_MODE = 'Translate';
+          $scope.TRANSLATION_MODE = 'Voiceover';
```

Comment:
I don't understand this change

Reply:
I had swapped the string value and it wasn't getting caught as this string value is used for switching mode and this will work with any value. I have fixed this!

(Done!)